benchmarks/chained-function-serving/driver/driver.go

```diff
@@ -204,12 +206,10 @@ func benchFanIn(ctx context.Context, prodAddr, consAddr string, fanInAmount int)
 	}
 	var capabilities []string
 	for i := 0; i < fanInAmount; i++ {
-		select {
-		case err := <-errorChannel:
-			if err != nil {
-				log.Errorf("[driver] FanIn push failed: %v", err)
-				return
-			}
+		err := <-errorChannel
```

Comment:
Won't the errorChannel block until it receives anything?

Reply:
You are right. I too want to revert this. The commitlint was failing due to this though. Shall I add an exception for this in the commitlint?

Reply:
what is the issue? does the test fail in that case?

Reply:
@shyamjesal I mean: does it fail with linter's suggestion?

Reply:
no. It worked before, it works now too.

Reply:
ok, let's make linter happy

Reply:
Sorry for the confusion.
I might did not correctly understood how it works. Probably my bad.
Now I see how it works. The loop waits until all coroutines return. So its meant to block until all respond.
Somehow I thought the previous code was non blocking.
Sorry, my bad.